kms/capi/no_capi.go

```diff
@@ -0,0 +1,20 @@
+//go:build !windows || nocapi
+// +build !windows nocapi
+
+package cloudkms
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"go.step.sm/crypto/kms/apiv1"
+)
+
+func init() {
+	apiv1.Register(apiv1.CAPIKMS, func(ctx context.Context, opts apiv1.Options) (apiv1.KeyManager, error) {
+		name := filepath.Base(os.Args[0])
+		return nil, errors.Errorf("unsupported kms type 'capi': %s is compiled without Microsoft CryptoAPI support", name)
```

Comment:
Does this need the call stack? I think `fmt.Errorf` is sufficient for those errors and removes unwanted dependency on `github.com/pkg/errors`

Reply:
That package was used in all the rest of the kms code, so I didn't change it.

Reply:
Then we're never going to get rid of it 😥

Reply:
If we want to get rid of it we'll probably should do it in PRs that cover one or more packages. At that time we will need to decide a few things, should we use more typed errors? Should we the `errors.Join`? Do we want to keep a stack trace?